src/controllers/likeController.js

```diff
@@ -0,0 +1,127 @@
+import * as ApplicationError from '../utils/errors/applicationsErrors';
+import { UpdateOrCreate, getLikes } from '../services/likeService';
+import { Accomodation, Like} from '../database/models';
+
+const like = async (req, res) => {
+  try {
+    const { accomodation_id: accomodationId } = req.params;
```

Comment:
Seems like everything looks okay. Only missing validation for the accomodationId. Which would fail if it's not provided when you try and destructure it here

Reply:
I have done middleware for validating parameters it is in idValidate.js